api/openapi-spec/swagger.json

```diff
@@ -7382,7 +7390,8 @@
         "ready",
         "restartCount",
         "image",
-        "imageID"
+        "imageID",
+        "started"
```

Comment:
this should be dropped now that the field is marked as omitempty

Reply:
indeed, `hack/update-openapi-spec.sh` did the trick